.github/workflows/chromatic.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - run: yarn workspace @department-of-veterans-affairs/web-components run build
       - run: yarn workspace @department-of-veterans-affairs/component-library run build
       - name: Publish to Chromatic
-        uses: chromaui/action@v1
+        uses: chromaui/action@latest
```

Comment:
We were getting [Chromatic build failures](https://github.com/department-of-veterans-affairs/component-library/actions/runs/12146849638/job/33871711595) on this PR and updating to the latest of the action seems to fix it. Are we cool merging this update as part of the dependabot PR?

cc @department-of-veterans-affairs/platform-design-system-fe